samples/csharp/end-to-end-apps/Forecasting-Sales/README.md

```diff
@@ -0,0 +1,347 @@
+  - This step is optional as the web app is already configured to use a pre-trained model. But you can create your own trained model and swap the pre-trained model with your own.
+
+## ML.NET Code Overview
+
+### Problem
+
+This problem is centered around country and product forecasting based on previous sales.
+
+### DataSet
+
+To solve this problem, two independent ML models are built that take the following datasets as input:  
+
+| Data Set | columns |
+|----------|--------|
+| **products stats**  | next, productId, year, month, units, avg, count, max, min, prev      |
+| **country stats**  | next, country, year, month, max, min, std, count, sales, med, prev   |
+
+[Explanation of Dataset](docs/Details-of-Dataset.md) - Goto this link for detailed information on dataset.
+
+### ML Tasks - [Regression](https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#regression) and Time Series
+
+The sample shows two different ML tasks and algorithms that can be used for forecasting:
+-  **Regression** using FastTreeTweedie Regression
+-  **Time Series** using Single Spectrum Analysis
+
+The **Regression** ML task is a supervised machine learning task that is used to predict the value of the **next** period (in this case the sales prediction) from a set of related features/variables. **Regression** works best with linear data.
+
+The **Time Series** ML task is an estimation technique that can be used to forecast **multiple** periods in the future. **Time Series** works well in scenarios that involve non-linear data where trends and patterns are difficult to distinguish.
+
+### Solution
+
+To solve this problem, first we will build the ML models by training each model on existing data. Next, we will evaluate how good it is. Finally, we will consume the model to predict sales.
+
+Note that the **Regression** sample implements two independent models to forecast linear data:
+- Model to predict product's demand forecast for the next period (month)
+- Model to predict country's sales forecast for the next period (month)
+
+The **Time Series** sample currently implements the product's demand forecast for the next **two** periods (months). The **Time Series** sample uses the same products as in the **Regression** sample so that you can compare the forecasts from the two algorithms.
+
+When learning/researching the samples, you can focus choose to focus on one of the algorithms/scenarios/models.
+
+![Build -> Train -> Evaluate -> Consume](docs/images/modelpipeline.png)
+
+#### 1. Load the Dataset
+
+Both the **Regression** and **Time Series** samples start by loading data using **TextLoader**. To use **TextLoader**, we must specify the type of the class that represents the data schema. Our class type is **ProductData**. 
+
+```csharp
+ public class ProductData
+    {
+        // The index of column in LoadColumn(int index) should be matched with the position of columns in the underlying data file.
+
```

Comment:
```suggestion
```

Reply:
Would recommend just removing the space between the comments since it may look like something is missing under the first comment/that the comments are not related.